pkg/pillar/nireconciler/genericitems/dnsmasq.go

```diff
@@ -692,7 +692,9 @@ func (c *DnsmasqConfigurator) addDNSHostFile(instanceName string,
 		return err
 	}
 	defer file.Close()
-	_, err = file.WriteString(fmt.Sprintf("%s	%s\n", entry.IP, entry.Hostname))
+	for _, ip := range entry.IPs {
+		_, err = file.WriteString(fmt.Sprintf("%s	%s\n", ip, entry.Hostname))
```

Comment:
Should we check error for each write?

Reply:
Agree, otherwise we are checking only last error. And how about changing real tab to \t symbol?

Reply:
Fixed the error check. Also replaced real tab with the escaped variant.